charts/rancher-monitoring/104.0.2+up45.31.1/files/rancher/pods/rancher-pod-containers.json

```diff
@@ -584,7 +584,7 @@
         "label": null,
         "multi": false,
         "name": "namespace",
-        "query": "label_values(kube_pod_info{cluster=\"$cluster\"}, namespace)",
```

Comment:
was the `cluster` labelpackaged with some older prometheus operator version, or is it a a custom rancher overlay, or is it a prometheus operator label we choose to not use

Reply:
From what I can tell its a label that is used by rancher when displaying metrics in the UI which is needed because rancher is multi-cluster. My guess is that when the queries were changed, they moved in the cluster variable either assuming it was the same thing, or left it there unintentionally.

We could have added the query used to get the cluster names (ie `label_values(up{job="kube-state-metrics"}, cluster)`), but why include an extra query that won't really be useful for a single cluster app.